bitex/interface/bitstamp.py

```diff
@@ -30,8 +26,13 @@ def __init__(self, **api_kwargs):
 
     def _get_supported_pairs(self):
         """Return a list of supported pairs."""
-        resp = requests.request('GET', 'https://www.bitstamp.net/api/v2/trading-pairs-info/')
-        return [pair["name"].replace("/", "").lower() for pair in resp.json()]
+        r = super(Bitstamp, self).request('GET',
+                                          'https://www.bitstamp.net/api/v2/trading-pairs-info/',
+                                          endpointwithversion=True)
+        if not r.ok:
+            raise PairFetchError(r.ok,r.status_code,r.reason)
+
+        return [pair["name"].replace("/", "").lower() for pair in r.json()]
```

Comment:
using requests seemed a bit more straightforward to me.

Reply:
but proxies cannot one point